### PR TITLE
Update pi-hole.css

### PR DESCRIPTION
**Pull request by XhmikosR**
* format code
* use `background-color` instead of the `background` shorthand
* use the `animation` shorthand
* ~~only transition opacity and transform in the chart tooltip~~
* add browserslist settings in package.json

**By submitting this pull request, I confirm the following:** `{please fill any appropriate checkboxes, e.g: [X]}`

`{Please ensure that your pull request is for the 'devel' branch!}`

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [ ] I have made only one major change in my proposed changes.
- [x] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

**This is untested**

It will conflict with #1094 so I'll need to rebase this later. We might also want to wait until we land #1080 and #1079 so that we do this once for the CSS part.

Non-whitespace diff: <https://github.com/pi-hole/AdminLTE/pull/1097/files?w=1>

### CODE REVIEW

File: style/pi-hole.css [hunk 1]
```diff
@@ -116,130 +155,135 @@
...
+  position: absolute;
+  background-color: rgba(0, 0, 0, .7);
+  color: #fff;
+  border-radius: 3px;
+  pointer-events: none;
+  -webkit-transition: opacity .1s ease, -webkit-transform .1s ease;
+  transition: opacity .1s ease, -webkit-transform .1s ease;
+  -o-transition: opacity .1s ease, -o-transform .1s ease;
+  transition: opacity .1s ease, transform .1s ease;
+  transition: opacity .1s ease, transform .1s ease, -webkit-transform .1s ease, -o-transform .1s ease;
```
Comment (XhmikosR) [inline style/pi-hole.css]: this is not a duplicate property and I used the browserslistrc we used in Bootstrap 3.x which is what's used in this project. Although arguably there are some pretty ancient browsers included.

**Comment by XhmikosR** [comment]
I think I'm gonna redo this after we all agree on some sensible browserslistrc. There's no point in supporting IE < 9 or even 10, or Opera Mini since these should not work already anyway due to JS.

**Comment by XhmikosR** [comment]
@dschaper can we agree on a more realistic browserslistrc? Then I will update this and remove any obsolete vendor prefixes from our CSS.

**Comment by dschaper** [comment]
Sure, @PromoFaux which IE do we still need?

**Comment by PromoFaux** [comment]
IE10 and 11, though, 10 becomes EOL at the end of this month. I can't imagine there are too many enterprise environments out there running <11.

**Comment by dschaper** [comment]
I think a min of 11 is good.

**Comment by XhmikosR** [comment]
IMHO we should target IE >= 10. IE 9 might also work, but I cannot test it personally.

Plus the normal evergreen browsers.

See https://browsersl.ist/?q=defaults%2C+IE+%3E%3D+10%2C+not+ie_mob+%3C%3D+11 and play with it to visualize the browsers.

**Comment by dschaper** [comment]
If 10 is supportable then great. I was operating under the assumption that 11 would be uncomfortable to shim in but 10 would be like ripping off a band-aid. 

If it's not to much more then 10 would be great.

**Comment by XhmikosR** [comment]
It should work, but I cannot confirm if the current devel branch works with IE 10. As long as we don't use new features, we should be fine.

**Comment by XhmikosR** [comment]
This is ready for review. The browserslist config I used is added in package.json. I can add the needed tools to automate this if you guys want later (postcss-cli and autoprefixer)

**Comment by XhmikosR** [comment]
Alright, I went ahead and added everything. Now everything is automated regarding CSS and JS, including CSS prefixes for our CSS.

**Comment by XhmikosR** [comment]
This is ready for review.

File: style/pi-hole.css [hunk 2]
```diff
@@ -116,126 +137,127 @@
...
+  opacity: 1;
+  position: absolute;
+  background-color: rgba(0, 0, 0, 0.7);
+  color: #fff;
+  border-radius: 3px;
+  pointer-events: none;
+  -webkit-transition: opacity 0.1s ease, -webkit-transform 0.1s ease;
+  transition: opacity 0.1s ease, -webkit-transform 0.1s ease;
+  transition: opacity 0.1s ease, transform 0.1s ease;
+  transition: opacity 0.1s ease, transform 0.1s ease, -webkit-transform 0.1s ease;
```
Comment (XhmikosR) [inline style/pi-hole.css]: These are the changes I'd like a confirmation from someone else too.

Comment (PromoFaux) [reply style/pi-hole.css]: Not 100% sure what I'm looking for, but it does not appear to have broken anything...

Comment (XhmikosR) [reply style/pi-hole.css]: I think I'll revert this part just to be safe for now.

**Comment by XhmikosR** [comment]
Rebased

**Comment by XhmikosR** [comment]
Rebased.

**Comment by XhmikosR** [comment]
@DL6ER maybe we should do this after #1252? BTW I can remove any npm packages added since the prefixes don't change really often. Or split that to a new patch and keep the CSS only changes here.

**Review by PromoFaux — DISMISSED** [review]

**Comment by PromoFaux** [comment]
@XhmikosR Is this one ready for review and merge?

**Comment by XhmikosR** [comment]
I think I need to do a couple more tests.

**Comment by XhmikosR** [comment]
@PromoFaux : I think this is finally ready. I split the patches a bit, tighten the browserslist settings and it should be good to go after one more tests round :)

**Review by PromoFaux — APPROVED** [review]

**Pull request merged**